terminado/tests/test_basic.py

```diff
@@ -0,0 +1,46 @@
+#
+# basic_tests.py -- Basic unit tests for Terminado
+#
+
+from __future__ import absolute_import, print_function
+
+import unittest
+from terminado import SingleTermManager, TermSocket
+import tornado
+import tornado.httpserver
+import tornado.httpclient
+import tornado.testing
+import logging
+import json
+
+class BasicTest(tornado.testing.AsyncHTTPTestCase):
+	def get_app(self):
+		return tornado.web.Application([
+					(r"/websocket/(\w+)", TermSocket,

```

Comment:
There's no need to use a URL component with SingleTermManager, it will be ignored. Only NamedTermManager uses a name from the URL.